subgraphs/the-graph/README.md

```diff
@@ -0,0 +1,61 @@
+# Uniswap Forks
```

Comment:
You should probably update this doc. Specifically, include the methodology on how we are defining and calculating TVL/Revenue etc

Reply:
Also maybe get some feedback on the methodology with protocol specialists to make sure we are all aligned

Reply:
README added and ready for review.